### PR TITLE
Use the toml implementation from standard lib.

### PR DESCRIPTION
**Pull request by sallner**
Unfortunately, the standard lib only supplies read-only functionality. `tomli-w` is a small `TOML` writing library. I should be enough for our purpose, but it cannot write comments

Using `tomlkit` would allow fine more finely grained editing, but comes with a whole new data structure and needs a bigger refactoring.

The changes in this PR will change the formatting of the .meta.toml slightly, see the result for the Zope package

<details>
<summary> Diff of the .meta.toml in Zope </summary>

```diff
diff --git a/.meta.toml b/.meta.toml
index 9a080cba6..c352105f5 100644
--- a/.meta.toml
+++ b/.meta.toml
@@ -2,7 +2,7 @@
 # https://github.com/zopefoundation/meta/tree/master/config/zope-product
 [meta]
 template = "zope-product"
-commit-id = "a3ddb354"
+commit-id = "77efeb4b"
 
 [python]
 with-pypy = false
@@ -26,23 +26,23 @@ known_zope = "AccessControl, Acquisition, App, DateTime, DocumentTemplate, Exten
 additional-config = [
     "# W503 line break before binary operator",
     "ignore = W503",
-    ]
+]
 
 [tox]
 testenv-deps = [
     "cffi",
-    ]
+]
 additional-envlist = [
     "pre-commit",
-    ]
+]
 testenv-commands-pre = [
     "{envbindir}/buildout -c {toxinidir}/buildout.cfg buildout:directory={envdir} buildout:develop={toxinidir} buildout:root-directory={toxinidir}",
-    ]
+]
 testenv-commands = [
     "# the `layer` argument below suppresses a `ZCatalog` performance test",
     "# which occasionally fails on GITHUB",
     "{envdir}/bin/alltests --layer '!Products.PluginIndexes' {posargs:-vc}",
-    ]
+]
 coverage-basepython = "python3.8"
 coverage-command = "coverage run {envdir}/bin/alltests {posargs:-vc}"
 testenv-additional = [
@@ -68,7 +68,7 @@ testenv-additional = [
     "commands =",
     "    autopep8 --verbose --in-place --recursive --aggressive --aggressive {toxinidir}/src setup.py",
     "    docformatter --in-place --recursive {toxinidir}/src setup.py",
-    ]
+]
 use-flake8 = true
 
 [manifest]
@@ -110,7 +110,7 @@ additional-rules = [
     "recursive-include src *.xml",
     "recursive-include src *.zcml",
     "recursive-include src *.zpt",
-    ]
+]
 
 [check-manifest]
 additional-ignores = [
@@ -123,13 +123,13 @@ additional-ignores = [
     "docs/_build/html/_sources/zopebook/includes/*",
     "docs/_build/html/_static/*",
     "docs/_build/html/_static/css/*",
-    ]
+]
 ignore-bad-ideas = [
     "src/Products/Five/tests/locales/de/LC_MESSAGES/fivetest.mo",
     "src/Products/Five/tests/locales/en/LC_MESSAGES/fivetest.mo",
-    ]
+]
 
 [git]
 ignore = [
     "docs/locale/*/LC_MESSAGES/*.mo",
-    ]
+]
```

</details>

### CODE REVIEW

**Comment by sallner** [comment]
This solves #118.

File: config/README.rst [hunk 1]
```diff
@@ -91,7 +91,8 @@ Usage
 Preparation
 +++++++++++
 
-The script needs a ``venv`` with some packages installed::
+The script needs a ``venv`` with at Python > 3.11 with some packages
```
Comment (dataflake) [inline config/README.rst:94]:
```suggestion
The script needs a ``venv`` with at least Python 3.11 and some dependencies
```

**Review by dataflake — COMMENTED** [review]
One small test change. I'll leave the actual review to Michael.

**Comment by dataflake** [comment]
My gut feeling is that if we spend time on switching TOML libraries then it makes more sense to invest a little more and switch to `tomlkit`. This PR uses a mixture of `tomli_w` and `tomllib` where `tomli_w` is a bit lacking in capabilities. Using `tomlkit` would also remove the Python 3.11 limitation imposed by the use of `tomllib`.

**Comment by dataflake** [comment]
See #215 for an alternative implementation that uses `tomlkit`

**Comment by icemac** [comment]
Superseded by #215.

Thank you for the initial implementation.

**Pull request closed without merging**